great_expectations/checkpoint/actions.py

```diff
@@ -439,32 +450,42 @@ def _run(  # type: ignore[override] # signature does not match parent  # noqa: P
             )
 
         validation_success = validation_result_suite.success
+        expectation_suite_name = validation_result_suite.meta.get(
+            "expectation_suite_name", "__no_expectation_suite_name__"
+        )
+
+        return self._run_pypd_alert(
+            name=expectation_suite_name,
+            template="Great Expectations suite check {name} has failed",
+            success=validation_success,
+        )
+
+    def _run_pypd_alert(self, name: str, template: str, success: bool):
```

Comment:
Well its either a suite name or a checkpoint name (depending on what code path we're serving here). I'd like to keep this as `name` as that is the name of the template param?

This will all go once we remove legacy checkpoints

Reply:
See comment above.